package.json

```diff
@@ -116,6 +116,7 @@
     "expo-av": "~13.10.4",
     "expo-image": "1.11.0",
     "expo-image-manipulator": "11.8.0",
+    "fast-equals": "^5.0.1",
```

Comment:
Interesting ... we were already using this in `OptionRowLHNData`, but it was not in our `package.json`. This leads me to believe it is a dependency of one of our existing packages.

After looking into it, I see it's used in `react-native-onyx`. However, we have a different major version of this package in `react-native-onyx` than here. I think the result is that we have to copies of `fast-equals` in E/App. Can we  create a PR to upgrade `fast-equals` in `react-native-onyx` too so that we end up with only one copy of this lib in E/App?

Reply:
Alternatively, just remove this and rely on it being a dependency of Onyx. But I don't think that's really a best practice...

Reply:
That's a sneaky one, nice catch @roryabraham! :D 

After stumbling upon it, it was reported [here](https://callstack-hq.slack.com/archives/C01GTK53T8Q/p1718732844064939) and @gedu already made an issue https://github.com/Expensify/App/issues/43997. 

After corresponding [PR](https://github.com/Expensify/App/pull/44018) is merged we can merge main into this branch to match versions with Onyx.

Reply:
Great, went ahead and merged https://github.com/Expensify/App/pull/44018